<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>630</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QStackedWidget" name="changeWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>-10</y>
      <width>951</width>
      <height>641</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 242, 242);
QStackedWidget::up-arrow,QStackedWidget::down-arrow{width: 0px;height: 0px;}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="messagePage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QSplitter" name="messageSplider">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>11</y>
        <width>941</width>
        <height>631</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,20">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <widget class="QLineEdit" name="searchLineEdit">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>211</width>
                <height>41</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit#searchLineEdit{
	border-radius:5px;
	border-with:0;
	font: 400 12pt &quot;微软雅黑&quot;;
	background-color: rgb(242, 242, 242);
}</string>
              </property>
             </widget>
             <widget class="QPushButton" name="searchButton">
              <property name="geometry">
               <rect>
                <x>230</x>
                <y>20</y>
                <width>39</width>
                <height>39</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>39</width>
                <height>39</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>39</width>
                <height>39</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#searchButton{
	border-radius:5px;
	border-with:0;
	font: 400 12pt &quot;微软雅黑&quot;;
	background-color: rgb(242, 242, 242);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../src.qrc">
                <normaloff>:/mainMenu/img/add_imge.svg</normaloff>:/mainMenu/img/add_imge.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>15</width>
                <height>15</height>
               </size>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="messageList">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QListWidget{
	border: none;
	background-color: rgb(255, 255, 255);
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="messagePageWidget" native="true"/>
     </widget>
    </widget>
    <widget class="QWidget" name="FriendListPage">
     <widget class="QSplitter" name="friendListSplider">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>10</y>
        <width>941</width>
        <height>631</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_7" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_6" stretch="3,20">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget_8" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <widget class="QLineEdit" name="searchLineEdit_3">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>211</width>
                <height>41</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit#searchLineEdit_3{
	border-radius:5px;
	border-with:0;
	font: 400 12pt &quot;微软雅黑&quot;;
	background-color: rgb(242, 242, 242);
}</string>
              </property>
             </widget>
             <widget class="QPushButton" name="searchButton_3">
              <property name="geometry">
               <rect>
                <x>230</x>
                <y>20</y>
                <width>39</width>
                <height>39</height>
               </rect>
              </property>
              <property name="minimumSize">
               <size>
                <width>39</width>
                <height>39</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>39</width>
                <height>39</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton#searchButton_3{
	border-radius:5px;
	border-with:0;
	font: 400 12pt &quot;微软雅黑&quot;;
	background-color: rgb(242, 242, 242);
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../src.qrc">
                <normaloff>:/mainMenu/img/add_imge.svg</normaloff>:/mainMenu/img/add_imge.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>15</width>
                <height>15</height>
               </size>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="messageList_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QListWidget{
	border: none;
	background-color: rgb(255, 255, 255);
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="friendListWidget" native="true"/>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="SiderBar" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>61</width>
      <height>631</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 242, 242);</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QToolButton" name="ListButton">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton#ListButton{
	border: none;
	image: url(:/mainMenu/img/friendListU.png);
	border-radius: 10px
}

QToolButton#ListButton:checked{
	image: url(:/mainMenu/img/FriendListO.png);
	background-color: rgb(233, 233, 233);
	border-radius: 10px
}

QToolButton#ListButton:hover{
	background-color: rgb(233, 233, 233);
	border-radius: 10px
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QToolButton" name="ConnectButton">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton#ConnectButton{
	border: none;
	image: url(:/mainMenu/img/collect.png);
	border-radius: 10px
}

QToolButton#ConnectButton:hover{
	image: url(:/mainMenu/img/collectO.png);
	border-radius: 10px
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QToolButton" name="FileButton">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton#FileButton{
	border: none;
	image: url(:/mainMenu/img/file.png);
	border-radius: 10px
}

QToolButton#FileButton:hover{
	image: url(:/mainMenu/img/fileO.png);
	border-radius: 10px
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QToolButton" name="ChannelButton">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton#ChannelButton{
	border: none;
	image: url(:/mainMenu/img/channel.png);
	border-radius: 10px
}

QToolButton#ChannelButton:hover{
	background-color: rgb(233, 233, 233);
	border-radius: 10px
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QToolButton" name="MenuButton">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton#MenuButton{
	border: none;
	image: url(:/mainMenu/img/normal_list_icon.f447ca40b46870868357.svg);
	border-radius: 10px
}

QToolButton#MenuButton:hover{
	image: url(:/mainMenu/img/normal_list_active_icon.a3c44178dd097a6f1cda.svg);
	border-radius: 10px
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="MessageButton">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton#MessageButton{
	border: none;
	image: url(:/button/img/normal_message.png);
	border-radius: 10px
}

QToolButton#MessageButton:checked{
	image: url(:/button/img/highlight_message.svg);
	background-color: rgb(233, 233, 233);
	border-radius: 10px
}

QToolButton#MessageButton:hover{
	background-color: rgb(233, 233, 233);
	border-radius: 10px
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <widget class="QToolButton" name="MailButton">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton#MailButton{
	border: none;
	image: url(:/mainMenu/img/mail.png);
	border-radius: 10px
}

QToolButton#MailButton:hover{
	image: url(:/mainMenu/img/emailO.png);
	border-radius: 10px
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QToolButton" name="RoomButton">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton#RoomButton{
	border: none;
	image: url(:/mainMenu/img/space.png);
	border-radius: 10px
}

QToolButton#RoomButton:hover{
	background-color: rgb(233, 233, 233);
	border-radius: 10px
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="userImage">
       <property name="minimumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>42</width>
         <height>42</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../src.qrc"/>
 </resources>
 <connections/>
</ui>
